scripts/variantstore/wdl/GvsPrepareCallset.wdl

```diff
@@ -62,6 +64,8 @@ task PrepareCallsetTask {
         String docker
     }
 
+    Array[String] query_label_args = if defined(query_labels) then prefix("--query_labels ", select_first([query_labels])) else []
```

Comment:
What does this do?  My reading is like:

IF we have query labels, then prepend "--query_labels" to the first query label 
ELSE return empty array

In the case of > 1 label, won't this lose everything but the first one?

Reply:
let's chat faces about this one!